crates/tessellation/src/fill.rs

```diff
@@ -174,11 +174,11 @@ struct ActiveEdges {
 }
 
 struct Span {
-    tess: Option<Box<MonotoneTessellator>>,
+    tess: Option<MonotoneTessellator>,
```

Comment:
This is actually undoing a performance optimization (see #621), let's roll it back.

Reply:
I've rolled it back, and added a doc comment referencing this issue so that it doesn't happen again.